tests/foreman/cli/test_import.py

```diff
@@ -315,14 +345,56 @@ def test_import_users_default(self, data):
         })
         os.remove(new_dataset)
         ssh.command(
-            'rm -rf ${{HOME}}/.transition_data {0} {1}'
-            .format(pwdfile, tmp_dir)
+            'rm -rf ${{HOME}}/.transition_data {}'
+            .format(pwdfile)
         )
         self.assertEqual(
             ssh_import.stdout,
             [u'Summary', u'  Created 3 users.', u'']
         )
 
+    @data(*positive_import_user_data())
+    def test_reimport_users_default(self, test_data):
```

Comment:
As far as I understand this is a negative test scenario, so:
1) You are not doing refactoring only, but adding coverage (maybe, it will be fine to rename PR/commit)
2) It is a negative test, but we pass here ```positive_import_user_data``` (just confused me)
3) It is not bad idea to add _negative suffix to the method name in order to run only positive or only negative tests in future